sky/services/platform/filesystem.mojom

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module filesystem;
```

Comment:
We should give these more esoteric module names.  I would have expected a filesystem module to be an actual file system (e.g., read/write/flush).

module flutter.platform

?